### ISSUE TITLE
[Adaptive Cards-More Option]: Visual name and accessible name defined for the 'More/Less (...)' icon button present on the header section of the application is different.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9214]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'More/Less (...)' icon button present on the header section of the application using tab key.
3. Verify that, descriptive accessible name is defined for the 'More/Less (...)' icon button present on the header section of the application or not.

**Actual Result:**
Visual name and accessible name defined for the 'More/Less (...)' icon button present on the header section of the application is different. Currently screen reader is only announcing 'More app bar button' and 'Less app bar button' which is not descriptive enough.

**Note**
Currently "See More" and "See Less" is defined as tooltip for the 'More/Less (...)' icon button.

**Expected Result:**
Visual name and accessible name defined for the 'More/Less (...)' icon button present on the header section of the application should be same. Screen reader should properly announce as "See More Options app bar button" and "See Less Options app bar button" which is same as per the tooltip information.

**User Impact:**
User who rely on screen reader to navigate on the application will be confused if descriptive accessible name is not defined for the 'More/Less (...)' icon button present on the header section of the application due to which user will not be easily able to identify the purpose of the control and might get confused on navigating the application.

WCAG Reference link:
https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/7d2f6475-344d-42f5-97d7-d47665cc9cb6)

https://github.com/user-attachments/assets/5a637b60-620a-4ef5-ad64-42cd8d5ab98f

### Card JSON

```JSON
N/A
```
